3-enrich/spark-enrich/README.md

```diff
@@ -1,14 +1,14 @@
-# Scala Hadoop Enrich
+# Spark Enrich
 
 ## Introduction
 
-This is the Snowplow Enrichment process implemented for Hadoop using [Scalding][scalding]. The Hadoop Enrichment parses raw Snowplow event logs, extracts the Snowplow events, enriches them (e.g. with geo-location information) and then writes them out to Snowplow-format event files.
+This is the Snowplow Enrichment process implemented using [Spark] [spark]. The Hadoop Enrichment parses raw Snowplow event logs, extracts the Snowplow events, enriches them (e.g. with geo-location information) and then writes them out to Snowplow-format event files.
```

Comment:
Please fix links according to CommonMark spec. There should be no spaces between brackets.